dim/dim/ldap_sync.py

```diff
@@ -106,38 +115,38 @@ def sync_users(ldap: LDAP, dry_run: bool = False):
                              (db_user.username,
                               db_user.ldap_cn,
                               ldap_user.ldap_cn))
-                if not dry_run:
-                    db_user.ldap_cn = ldap_user.ldap_cn
+                db_user.ldap_cn = ldap_user.ldap_cn
             if db_user.department_number != ldap_user.department_number:
                 logging.info('User %s moved from department_number %s to %s' %
                              (db_user.username,
                               db_user.department_number,
                               ldap_user.department_number))
-                if not dry_run:
-                    db_user.department_number = ldap_user.department_number
+                db_user.department_number = ldap_user.department_number
             if db_user.ldap_uid != ldap_user.ldap_uid:
                 logging.info('User %s changed uid from %s to %s' %
                              (db_user.username,
                               db_user.ldap_uid,
                               ldap_user.ldap_uid))
-                if not dry_run:
-                    db_user.ldap_uid = ldap_user.ldap_uid
+                db_user.ldap_uid = ldap_user.ldap_uid
         elif db_user.ldap_uid:
             log_stdout('Deleting user %s' % db_user.username)
-            if not dry_run:
-                db.session.delete(db_user)
+            db.session.delete(db_user)
+    if not ignore_deletion_threshold:
+        check_deletion_threshold(User, deletion_threshold)
 
 
 @time_function
 @transaction
-def ldap_sync(dry_run: bool = False):
+def ldap_sync(ignore_deletion_threshold: bool = False):
```

Comment:
The dry_run option was removed here, but is still called in manage_dim. Is that correct?